docs/quickstart.md

```diff
@@ -2,6 +2,8 @@
 
 To quickly get started using ModelMesh Serving, here is a brief guide.
 
+> **Note**: To install the latest stable release, please follow the [Quick Start Guide for version 0.11](https://github.com/kserve/modelmesh-serving/blob/release-0.11/docs/quickstart.md).
```

Comment:
I think this note is good, but we should preface it by saying that the instructions in this guide are for the `latest` / `main` branch ("under development").

A bit further down we should change `RELEASE=release-0.10` to `RELEASE=main` and when we draft a release we replace it with the latest new release branch. And add a sentence that this is only necessary of the repo has not been (forked and) cloned yet.


we probably need the same kind of note in other install docs (`docs/install/README.md`, FVT, ...) where ever we ask to clone the repo and checkout a `release-0.10` or `release-0.11` branch

Reply:
Hi @ckadner,
So sorry for the late response, let me get to your suggestions!

> preface it by saying that the instructions in this guide are for the latest / main branch ("under development").

What do you think about adding this sentence: 

```
The following instructions are only for early adopters who want to install the latest ModelMesh release which is under development. To install the latest stable release...
```

> change RELEASE=release-0.10 to RELEASE=main and when we draft a release we replace it with the latest new release branch. 

Sounds good, so v0.10 the latest stable release, and I should change the link to [Quick Start Guide for version 0.10](https://github.com/kserve/modelmesh-serving/blob/release-0.10/docs/quickstart.md). And keep it updated somehow in the future. (Perhaps with the usage of conrefs/keydefs...)

> we probably need the same kind of note in other install docs (docs/install/README.md, FVT, ...) where ever we ask to clone the repo and checkout a release-0.10 or release-0.11 branch

a. I see README.md already links to the Quick Start Guide at the bottom
b. What do you think of adding this kind of note for FVT docs and other varieties: 

```
The following instructions are only for early adopters who want to install the latest ModelMesh release with Gingko which is under development. To install the latest stable release...
```

@rafvasq

Reply:
> The following instructions are only for early adopters who want to install the latest ModelMesh release which is under development. To install the latest stable release...

I like this but maybe shorten to `The following instructions install the latest ModelMesh release which is under development. To install the latest stable release...`

> a. I see README.md already links to the Quick Start Guide at the bottom

Just to be clear in case you're referring to this [README](https://github.com/kserve/modelmesh-serving/blob/main/docs/README.md), I don't think it needs to be changed. I believe @ckadner is referring to the parts of docs such as [this one](https://github.com/kserve/modelmesh-serving/blob/main/docs/install/install-script.md#installation-1), which include explicit instructions for installation. Here, I believe we would keep it pointed to the latest official release (v0.10.0, soon to be v0.11.0).

Reply:
And the code snippet below:

> ### Get the latest release
> ```shell
> RELEASE=release-0.10
> git clone -b $RELEASE --depth 1 --single-branch https://github.com/kserve/modelmesh-serving.git
> cd modelmesh-serving
> ```

Needs to be updated to:

### Clone the ModelMesh repository
```shell
RELEASE=main
git clone -b $RELEASE --depth 1 --single-branch https://github.com/kserve/modelmesh-serving.git
cd modelmesh-serving
```